charts/yurt-manager/templates/yurt-manager-auto-generated.yaml

```diff
@@ -194,6 +200,7 @@ rules:
   - nodebuckets
   - nodepools
   - yurtappdaemons
+  - yurtappoverriders
```

Comment:
I think `yurtappoverriders` has already been removed, why we need to add it here?

Reply:
done